email/track-editing-allotment.html

```diff
@@ -26,22 +26,22 @@
     <tbody valign="top">
       {% for task in params.tasks %}
       <tr>
-        <td>{{ task.outputFileLink }}</td>
+        <td>{{ task.fileName }}</td>
         <td>
-          {% for chunk in task.chunks %}
-          {{ chunk.fileName }};&nbsp;{{ chunk.beginning }}-{{ chunk.ending
+          {% for chunk in task.task.chunks %}
```

Comment:
I think this duplication `task.task` can be avoided by injecting `fileName` into the original task obtained from the database.